core/arch/arm/kernel/thread_spmc.c

```diff
@@ -431,12 +488,14 @@ void spmc_handle_partition_info_get(struct thread_smc_args *args,
 	}
 
 	ret_fid = FFA_SUCCESS_32;
-	rxtx->tx_is_mine = false;
+	if (!count_only)
+		rxtx->tx_is_mine = false;
```

Comment:
I think this should be moved after the `out` label. From the `if (is_nil_uuid...` case we always jump to `out` which means the updating of `tx_is_mine` is skipped. I'm not sure why this didn't cause any errors before, but it looks like this is the reason for SPMC test failure I was getting.

Reply:
Makes sense, thanks for looking into this.